Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -445,4 +445,23 @@ class BrowserCoordinator: BaseCoordinator,
             findAndHandle(route: savedRoute)
         }
     }
+
+    func showTabTray() {
+        if let tabTrayCoordinator = childCoordinators[TabTrayCoordinator.self] {
+            tabTrayCoordinator.start()
+            (tabTrayCoordinator.router.navigationController as? UINavigationController).map { router.present($0) }
```

Comment:
I think we can simplify this to the equivalent of
```
        guard !childCoordinators.contains(where: { $0 is FakespotCoordinator}) else {
            return // flow is already handled
        }
```
Basically if `showTabTray` is asked twice and tab tray coordinator was still there, it means the coordinator is already showing the tab tray since it wasn't deinit. So we should have to do anything or reshow the coordinator views.